src/kolibri_gnome/application.py

```diff
@@ -128,7 +128,7 @@ def open_kolibri_window(
     ) -> typing.Optional[KolibriWindow]:
         target_url = target_url or self.__context.default_url
 
-        if not self.__context.should_load_url(target_url):
+        if not self.__context.should_open_url(target_url):
```

Comment:
Yes, this rename is great, much clear as you say in the commit message. I also like to rename things on the go as I change code. Is a good way to keep the codebase in shape.